Console/Command/SyncProductCategoriesCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Taxjar
+ * @package    Taxjar_SalesTax
+ * @copyright  Copyright (c) 2020 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc. (http://www.taxjar.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace Taxjar\SalesTax\Console\Command;
+
+use Magento\Framework\App\State;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Taxjar\SalesTax\Model\Logger;
+use Taxjar\Salestax\Observer\ImportCategories;
+
+class SyncProductCategoriesCommand extends Command
+{
+    /**
+     * @var State
+     */
+    protected $state;
+
+    /**
+     * @var Logger
+     */
+    protected $logger;
+
+    /**
+     * @var ImportCategories
+     */
+    protected $importCategories;
+
+    /**
+     * @param State $state
+     * @param Logger $logger
+     * @param ImportCategories $importCategories
+     */
+    public function __construct(
+        State $state,
+        Logger $logger,
+        ImportCategories $importCategories
+    ) {
+        $this->state = $state;
+        $this->logger = $logger;
+        $this->importCategories = $importCategories;
+
+        parent::__construct();
+    }
+
+    /**
+     * Sets config for CLI command
+     */
+    protected function configure()
+    {
+        $this->setName('taxjar:product_categories:import')
```

Comment:
@ryanreeves-taxjar Consider changing this command to `taxjar:product_categories:sync` similar to `taxjar:transactions:sync`. We use "Sync from TaxJar" on the button, so I think it makes sense to use "sync" as a verb for the CLI as well.

Reply:
Good catch!